shared/api/network_acl.go

```diff
@@ -0,0 +1,52 @@
+package api
```

Comment:
Aren't the field descriptions going to be redundant with what we'll have in `doc/`?

If so, we may want to strip them from `shared/api` just to avoid them getting out of sync as we add new supported values down the line.

Eventually we'll want to handle all this stuff through #7874 but until then, best to not duplicate.

Reply:
fixed